src/org/exoplatform/ui/DashboardActivity.java

```diff
@@ -133,15 +133,23 @@ private void initStubView() {
   }
 
   @Override
-  public boolean onHandleActionBarItemClick(ActionBarItem item, int position) {
-    switch (position) {
+  public boolean onCreateOptionsMenu(Menu menu) {
+    getMenuInflater().inflate(R.menu.dashboard, menu);
+    loaderItem = menu.findItem(R.id.menu_as_refresh);
+    return true;
+  }
+  
+  @Override
+  public boolean onOptionsItemSelected(MenuItem item) {
+
+    switch (item.getItemId()) {
 
-    case -1:
+    case android.R.id.home:
       finish();
```

Comment:
I think we can remove the entire `case home: ... break;` as it's already the default behavior of the up navigation button.